gestore/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 # Increase this version by 1 after every backward-incompatible
 # change in the exported data format
-__version__ = '0.1.0-dev3'
+__version__ = '0.1.0.dev3'
```

Comment:
Trying again with semver.

```suggestion
__version__ = '0.1.dev3'
```